### PR TITLE
Nature VPAT 2.2 update

### PR DESCRIPTION
**Pull request by hollsk**
* Redirects all the individual product VPATs to the new combined Nature one
* Adds the new format. Best viewed in the file view: https://github.com/springernature/vpat/blob/3bfd1f43237001a162913f9ba4ea07d7498e4468/nature.md

### CODE REVIEW

File: nature.md [hunk 1]
```diff
@@ -0,0 +1,1441 @@
...
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially supported</td>
```
Comment (davidpauljunior) [inline nature.md]: There's a mix of `partially supports` and `partially-supported`.
The  [terms](https://github.com/springernature/vpat/pull/16/files#diff-996194ccbe898dcc45395e96e9ad8a03R42) suggest `partially supports` is the correct phrasing.

Comment (hollsk) [reply nature.md]: good spot, ta. changed all the "not supported"s to "Does Not Support" as well 👍

File: nature.md [hunk 2]
```diff
@@ -0,0 +1,1441 @@
...
+<br><br>
+Background images such as icons on user interface elements are generally identified with text labels. These may be visible, visually-hidden, or supplied using WAI-ARIA techniques.
+<br><br>
+We mark up some images so that they can be ignored by Assistive Technologies if they are <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration"><i>pure decoration</i></a> (as defined by WCAG 2.0).
+<br><br>
+The Nature subscription pages at <a href="https://www.nature.com/nature/subscribe">https://www.nature.com/nature/subscribe</a> currently contain SVG images without text alternatives, and decorative SVG images that are not marked up to be ignored by Assistive Technologies. We are actively working to repair these problems. 
+<br><br>
+Third party scripts may inject non-text content without text alternatives. These include images in advertising scripts, in Disqus commenting widgets, and in the "manage cookies" privacy centre. Where we know of deficiencies, we have contacted third party suppliers to request repairs.
```
Comment (davidpauljunior) [inline nature.md]: You specifically mention Disqus here.  Should we also specifically mention OneTrust as the manage cookies thing?

File: nature.md [hunk 3]
```diff
@@ -0,0 +1,1441 @@
...
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Content supplied by us does not play automatically. All audio files supplied by us, either hosted on our own platform or through a third party, has mechanisms for pausing, stopping, and controlling volume.
```
Comment (davidpauljunior) [inline nature.md]: has -> have

File: nature.md [hunk 4]
```diff
@@ -0,0 +1,1441 @@
...
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Generally, the purpose of each link that we provide can be determined by its descriptive text label; linked images or icons have alternative text or similar to provide this context. 
+<br><br>
+In research articles, anchor links that refer to same-page supplementary figures or tables may have insufficient context in their link text. Repetitive links (e.g. "show more" or "view all" type links in lists of content) may not be differentiated from each other. In limited cases we may support poor link text with the use of ARIA labels. We are actively working to repair these problems. 
+<br><br>
+Third party scripts including advertising scripts, the Disqus commenting widget, or survey or customer services widgets (among others), may inject content that includes links with no accessible text, or poorly-named accessible text. Where we know of deficiencies, we have contacted third party suppliers to request repairs.
```
Comment (davidpauljunior) [inline nature.md]: Do you think we should always use the same third party list?
When we mention 3rd parties throughout the document we sometimes mention disqus, sometimes ads, sometimes others.

Comment (hollsk) [reply nature.md]: IMO, no. For each criterion, I've tried to only mention the third party scripts that we either know to have problems in that area, or that I think have the real potential to create problems. 

There will always be unknowns when it comes to third parties, but I'm wary of hedging too much. It's true that third parties can potentially inject all kinds of problematic content, but we'd want to deal with that as it arises. In the meantime our customers need to make decisions on whether or not they can buy our software based on this document, so the main focus needs to be accuracy (both from the customer's perspective and from our business' perspective).

**Comment by moddular** [comment]
Looks good to me 👍

**Comment by nickcall** [comment]
Looks excellent 👍

**Comment by dotcode** [comment]
Thanks all very much for your work on this.

**Pull request merged**